src/documentation/print-linting-and-testing-wiki.ts

```diff
@@ -151,9 +165,8 @@ with the [functionality tests](#functionality-tests)).
 
 Have a look at the [test/system-tests](${RemoteFlowrFilePathBaseRef}test/system-tests) folder for more information.
  
-
-
-### Performance Tests
+<a id='performance-tests'></a>
```

Comment:
I think we need a function to abstract this away - maybe even a `section` function that does this for the respective level - just to make all sections more robust. What do you think?

Reply:
Would be nice to have! I'll open an issue :)

Reply:
#1606